### PR TITLE
Arcade Prize Additions

### PR DESCRIPTION
**Pull request by ada-please**
## About the PR
Added several items to the possible rewards list and rearranged list to be alphabetized to the first letter.

## Why / Balance
Its arcade prizes so I don't think it will affect balance much, if at all. I don't often see people play Space Villain but I get tired of receiving 3 griffin figures in a row, hopefully increasing the number of prizes will help with variety.

## Technical details
just a yml edit

## Media
NA

## Requirements

- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
none

**Changelog**

:cl:
- add: Added more prizes to the Space Villain arcade machine

### CODE REVIEW

**Comment by ps3moira** [comment]
Pretty sure the singulo toy causes server lag and that's why it's reserved as a admeme item.

**Comment by ada-please** [comment]
> Pretty sure the singulo toy causes server lag and that's why it's reserved as a admeme item.

i'd believe that, it was behaving a little odd during testing. I'll remove it

**Comment by IProduceWidgets** [comment]
This should probably be reworked to use entitytables

**Comment by Cojoke-dot** [comment]
Mmm... plushie throngler should only be loto... I like it being super rare

**Comment by ada-please** [comment]
> Mmm... plushie throngler should only be loto... I like it being super rare

I'll remove that one too

**Comment by ada-please** [comment]
> > Pretty sure the singulo toy causes server lag and that's why it's reserved as a admeme item.
> 
> i'd believe that, it was behaving a little odd during testing. I'll remove it

thought you said tesla for some reason. 
I'd be fine removing the singulotoy and the telsa but I like to keep them if it doesnt cause problems.

File: Resources/Prototypes/Entities/Structures/Machines/Computers/arcades.yml [hunk 1]
```diff
@@ -72,6 +72,59 @@
...
+    - PlushieSlime
+    - PlushieGhost
+    - PlushieDiona
+    - PlushieSharkBlue
+    - PlushieVox
+    - PlushieXeno
+    - PlasticBanana
+    - RevolverCapGun
+    - SnapPopBox
+    - TeslaToy
```
Comment (UbaserB) [inline Resources/Prototypes/Entities/Structures/Machines/Computers/arcades.yml:127]: remove this one probably

**Comment by UbaserB** [comment]
also it might be good to split it up so that there's a 50% chance to get a plushie and 50% chance to get a "toy" or something similar

File: Resources/Prototypes/Entities/Structures/Machines/Computers/arcades.yml (hunk 1 above)
Comment (ada-please) [reply Resources/Prototypes/Entities/Structures/Machines/Computers/arcades.yml:127]: you want the tesla gone too?

**Comment by ada-please** [comment]
> also it might be good to split it up so that there's a 50% chance to get a plushie and 50% chance to get a "toy" or something similar

I had considered switching the plushie spawns to a new "random plushie" item or something, same with figures (maybe only have it spawn the mystery box instead of single figures, but its was a quick PR so reworking it with entity tables and weighted spawning so could happen in the future.

File: Resources/Prototypes/Entities/Structures/Machines/Computers/arcades.yml (hunk 1 above)
Comment (ada-please) [reply Resources/Prototypes/Entities/Structures/Machines/Computers/arcades.yml]: just the tesla or all 4 or them?

Comment (ada-please) [reply Resources/Prototypes/Entities/Structures/Machines/Computers/arcades.yml]: I removed the tesla, but a cupgun, snap pops, and a plastic banana are all things ive received from real life arcades so I think they'd be appropriate to keep. also the capgun was already a reward on the list.

Comment (UbaserB) [reply Resources/Prototypes/Entities/Structures/Machines/Computers/arcades.yml]: > I removed the tesla, but a capgun, snap pops, and a plastic banana are all things ive received from real life arcades so I think they'd be appropriate to keep. also the capgun was already a reward on the list.

thats fine it was just the tesla

Comment (IProduceWidgets) [reply Resources/Prototypes/Entities/Structures/Machines/Computers/arcades.yml]: The plastic banana is a plastic explosive iirc

Comment (UbaserB) [reply Resources/Prototypes/Entities/Structures/Machines/Computers/arcades.yml]: > The plastic banana is a plastic explosive iirc

you can get it from toy crate so idk

Comment (ada-please) [reply Resources/Prototypes/Entities/Structures/Machines/Computers/arcades.yml]: I think this is the correct ID for the actual plastic toy banana. I think the explosive banana ID is something like explosivebanana but I dont rememeber

**Review by UbaserB — APPROVED** [review]
LGTM

**Review by slarticodefast — APPROVED** [review]
Good to merge when the conflict gets addressed.

**Comment by superjj18** [comment]
> Mmm... plushie throngler should only be loto... I like it being super rare

Winning that broken ass arcade machine is super rare :trollface:

**Comment by slarticodefast** [comment]
You resolved the pull request wrong: You removed the PlushieRainbowLizard again, which was added as an arcade prize in #32564

**Comment by slarticodefast** [comment]
> whoops, its my first time having a merge conflict so I just went with the section it highlighted, didnt notice the rainbow lizard. If I add it back in is everything good to go?

Yeah, if you add it back I can merge this PR. The addition of the rainbow lizard was what caused the merge conflict: Two Pull requests modifying the same part of the code in different ways.

**Comment by ada-please** [comment]
> > whoops, its my first time having a merge conflict so I just went with the section it highlighted, didnt notice the rainbow lizard. If I add it back in is everything good to go?
> 
> Yeah, if you add it back I can merge this PR. The addition of the rainbow lizard was what caused the merge conflict: Two Pull requests modifying the same part of the code in different ways.

Oh I see now, ok cool. 
Do I just add it back in with a new commit? or is there a way to roll it back then resolve the conflict properly?

**Comment by slarticodefast** [comment]
Just adding it back in a new commit is fine. When we merge a PR we squash it first, which means all your commits will be turned into a single one for better overview.

**Review by slarticodefast — APPROVED** [review]

**Comment by slarticodefast** [comment]
Looks good to me. Thank you for your contribution!

**Pull request merged**